examples/2016-10-31/usage_plan/template.yaml

```diff
@@ -0,0 +1,96 @@
+Conditions:
+  C1:
+    Fn::Equals:
+      - true
+      - true
```

Comment:
I don't think we need this condition in the examples, but it's great to include in the tests.